server/controllers/authUserController.js

```diff
@@ -0,0 +1,25 @@
+import importDeleteQuery from '../models/interactWithDatabase';
+
+class AuthUsers {
+  async logout(req, res) {
+    try {
+      if (req.headers.authorization) {
+        await importDeleteQuery.deleteValidToken(req.headers.authorization);
+        return res
+          .status(200)
+          .json({ status: 200, message: ` Hey Joshua !! you are logged out successfully` });
+      }
+      return res
```

Comment:
Why returning two times??

Reply:
For the first will be returned in the case token detected from a header and at the second will be returned when there is no token detected from a header.

Reply:
I corrected this mistake after analysing the useless algorithm I used before, now it looks great